core/tauri/src/updater/core.rs

```diff
@@ -715,9 +719,11 @@ fn copy_files_and_run<R: Read + Seek>(
     // If it's an `exe` we expect an installer not a runtime.
     if found_path.extension() == Some(OsStr::new("exe")) {
       // Run the EXE
-      Command::new(found_path)
+      Command::new("cmd")
+        .args(&["/C", "start", "", found_path.as_os_str().to_str().unwrap()])
```

Comment:
could you change this to use `powershell` instead? you will need to determine powershell path using `SYSTEMROOT` env var, move line 794 to line 715 and then do something similar  to line 795

Reply:
Don't worry about it, thanks for your contribution